<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Start_Btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>이름</string>
    </property>
   </widget>
   <widget class="QLabel" name="Phone_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>전화번호</string>
    </property>
   </widget>
   <widget class="QLabel" name="Mail_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>메일주소</string>
    </property>
   </widget>
   <widget class="QLabel" name="Number_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>생년월일</string>
    </property>
   </widget>
   <widget class="QLabel" name="CarNumber_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>차량번호</string>
    </property>
   </widget>
   <widget class="QLabel" name="Date_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>날짜</string>
    </property>
   </widget>
   <widget class="QLabel" name="Zone_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>위치</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_Input">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Phone_Input">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Mail_Input">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="CarNumber_input">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Number_Input">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="Camping">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>평화캠핑존</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="CaravanA">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>카라반존A</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="CaravanB">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>카라반존B</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ZoneNumber">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Adult_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>성인</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Adult_Input">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Day_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>기간</string>
    </property>
   </widget>
   <widget class="QLabel" name="Children_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>소인</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Children_Input">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="Day_One">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1박2일</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Day_Two">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>170</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2박3일</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Date_Input">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>689</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
